pkg/kubelet/apis/config/types.go

```diff
@@ -679,3 +685,11 @@ type MemorySwapConfiguration struct {
 	// +optional
 	SwapBehavior string
 }
+
+type CrashLoopBackOffConfig struct {
+	// maxSeconds is the maximum number of seconds the backoff delay can accrue
+	// to for container restarts
+	// +featureGate=EnableKubeletCrashLoopBackoffMax
+	// +optional
+	MaxSeconds *int32
```

Comment:
I know it's deviating from the KEP, but what do you think about changing this to a duration instead? Something like:
```suggestion
	MaximumBackoffPeriod *metav1.Duration
```

Reply:
I see a lot of Duration in this API so it SGTM

Reply:
Fixed in 6692dda